tests/http/conftest.py

```diff
@@ -102,7 +102,8 @@ def httpd(env) -> Generator[Httpd, None, None]:
         pytest.skip(f'httpd not found: {env.httpd}')
     httpd.clear_logs()
     if not httpd.start():
-        pytest.fail(f'failed to start httpd: {env.httpd}')
+        httpd.stop()
+        assert httpd.start(), f'failed to start httpd: {env.httpd}'
```

Comment:
I'm assuming this start/stop/start sequence is deliberate.

Reply:
Yes, if there is at the start of the fixture a httpd running, it is a leftover from a previous run most likely. Just try to get rid of it and attempt another start - for convenience.